openmetadata-ui/src/main/resources/ui/src/utils/CSV/CSVUtilsClassBase.tsx

```diff
@@ -309,6 +310,27 @@ class CSVUtilsClassBase {
             />
           );
         };
+      case 'entityType*':
+        return ({ value, ...props }) => {
+          const handleChange = (typeValue: string) => {
+            props.onChange(typeValue);
+          };
+
+          return (
+            <InlineEdit
+              className="w-full"
+              onCancel={props.onCancel}
+              onSave={props.onComplete}>
+              <Select
+                options={ENTITY_TYPE_OPTIONS}
+                size="small"
+                style={{ width: '155px' }}
```

Comment:
Instead of direct style please use class

Reply:
Noted, will be pushing this in separate PR!